common/test/acceptance/pages/studio/settings_certificates.py

```diff
@@ -57,6 +56,18 @@ def get_first_signatory_title(self):
         """
         return self.q(css='.signatory-title-value').first.html[0]
 
+    def get_course_number(self):
+        """
+        Return Course Number for the displayed certificate
```

Comment:
I think, course number is not related to certificate as it is the part of course. what you think if we update the doc string : 
`Return the course number`